frontend/device/gesturedetector.lua

```diff
@@ -440,10 +466,11 @@ function GestureDetector:handleSwipe(tev)
     logger.dbg("swipe", swipe_direction, swipe_distance, "detected in slot", slot)
     self:clearState(slot)
     return {
-        ges = "swipe",
+        ges = ges,
         -- use first pan tev coordination as swipe start point
         pos = start_pos,
         direction = swipe_direction,
+        multiswipe_directions = multiswipe_directions,
```

Comment:
@poire-z Perhaps this should just be `direction`. My original concept was to sneak multiswipes into regular swipes, but the kind of gestures that are broken by this are actually bugs anyway.

(E.g., for a "long diagonal swipe" you can swipe by two edges in what's now properly detected as a "multiswipe".)

Reply:
That is, the ges = ges line was something I changed last minute before creating the PR.

Reply:
Well, I don't have much of an opinion, I've never gone deep enough in the gesture code.
Question: two same-direction swipes (with a slight angle variation) still continue being a single swipe? or can you get multiswipe southeast_southeast?

Reply:
You can give it a go with diagonal multigestures but I found it a bit error-prone. But perhaps I still had the threshold too small then.

There's the line `not pan_direction:match(msd_direction_prev)` which matches `southeast` to `south` or `east` (whichever the currently going direction is). So only something completely different will go through at all.

Then later in `if msd_direction ~= msd_direction_prev then` its' ensured that the same direction isn't added twice.

To try diagonal multigestures you'd need to get rid of the "simple" path and that :match line.

Reply:
I guess we don't really need to support diagonal multi gestures, and I somehow think it's best not to, to ease distinguish them and avoid mis-decisions.